docs/en/week08/08-3.md

```diff
@@ -16,7 +16,6 @@ To summarize at a high level, a very simple form of AE is as follows:
 
 *For a detailed explaination, refer to the notes of [Week 7]({{site.baseurl}}/en/week07/07-3/).*
 
-
```

Comment:
This one was fine. We have two empty lines before any heading. Plz revert.

Reply:
Done